src/includes.ts

```diff
@@ -0,0 +1,60 @@
+import { purry } from './purry';
+import { _reduceLazy } from './_reduceLazy';
+import { equals } from './equals';
+
+export type Includable<T> = T[] | Object | string;
```

Comment:
I'm not sure if it should support both object and string. As far as I saw all the other functions support either object OR string, so it's a bit inconsistent wit the rest of the lib.

Reply:
Idk either - so what should `includes` handle? Only objects and arrays? Will there be a separate function for checking strings?